SimPEG/EM/TDEM/RxTDEM.py

```diff
@@ -0,0 +1,229 @@
+import SimPEG
+from SimPEG import Utils
+import scipy.sparse as sp
+
+
+class BaseRx(SimPEG.Survey.BaseTimeRx):
+    """
+    Time domain receiver base class
+
+    :param numpy.ndarray locs: receiver locations (ie. :code:`np.r_[x,y,z]`)
+    :param numpy.ndarray times: times
+    :param string orientation: receiver orientation 'x', 'y' or 'z'
+    """
+
+    def __init__(self, locs, times, orientation=None):
+        assert(orientation in ['x', 'y', 'z']), (
+            "Orientation {0!s} not known. Orientation must be in "
+            "'x', 'y', 'z'. Arbitrary orientations have not yet been "
+            "implemented.".format(orientation)
+        )
+        self.projComp = orientation
+        SimPEG.Survey.BaseTimeRx.__init__(self, locs, times, rxType=None) #TODO: remove rxType from baseRx
+
+    def projGLoc(self, f):
+        """Grid Location projection (e.g. Ex Fy ...)"""
+        return f._GLoc(self.projField) + self.projComp
+
+    def projTLoc(self, f):
+        """Time Location projection (e.g. CC N)"""
+        return f._TLoc(self.projField)
+
+    def getSpatialP(self, mesh, f):
```

Comment:
likely, this is a hangover from the first implementation I believe. I think this can be cleaned up later on if you are good with that